recipes_source/android_native_app_with_custom_op.rst

```diff
@@ -647,25 +642,25 @@ Now we are ready to write a native part of our application.
   }
 
 
-This listing is quite long, and a few things intermixed here, we will follow control flow to understand how this code works.
-The first place where the control flow arrives is ``JNI_OnLoad``.
-This function is called after loading the library. It is responsible for registering native method, which is called when ``NativePeer#loadAndForwardModel`` called, here it is ``pytorch_nativeapp::loadAndForwardModel`` function.
+이 목록은 꽤 길고 여기에 몇 가지가 혼합되어 있으므로 제어 흐름을 따라이 코드가 작동하는 방식을 이해할 것입니다.
+제어 흐름이 도착하는 첫 번째 위치는``JNI_OnLoad``입니다.
+이 함수는 라이브러리를로드 한 후 호출됩니다. ``NativePeer # loadAndForwardModel`` 이 호출 될 때 호출되는 네이티브 메서드를 등록하는 역할을 합니다. 여기서는 ``pytorch_nativeapp :: loadAndForwardModel`` 함수입니다.
 
-``pytorch_nativeapp::loadAndForwardModel`` takes as an argument model path.
-First, we extract its ``const char*`` value and loading the module with ``torch::jit::load``.
+``pytorch_nativeapp :: loadAndForwardModel`` 은 인수 모델 경로로 사용됩니다.
+먼저 ``const char *`` 값을 추출하고``torch :: jit :: load`` 로 모듈을 로드합니다.
 
-To load TorchScript model for mobile, we need to set these guards, because mobile build doesn't support 
-features like autograd for smaller build size, placed in ``struct JITCallGuard`` in this example.
-It may change in the future. You can track the latest changes keeping an eye on the 
-`source in PyTorch GitHub <https://github.com/pytorch/pytorch/blob/master/android/pytorch_android/src/main/cpp/pytorch_jni_jit.cpp>`_.
+모바일 용 TorchScript 모델을로드하려면 모바일 빌드가 지원하지 않기 때문에 이러한 가드를 설정해야합니다.
+이 예제에서는 ``struct JITCallGuard '' 에 배치 된 더 작은 빌드 크기를위한 autograd와 같은 기능입니다.
```

Comment:
To load TorchScript model for mobile, we need to set these guards, because mobile build doesn't support features like autograd for smaller build size, placed in ``struct JITCallGuard`` in this example. 를 한 문장으로 해석하는게 맞는 것 같습니다.